### PR TITLE
New & Updated materialized themes

### PR DESCRIPTION
**Pull request by MatiDragon-YT**

### CODE REVIEW

**Comment by x87** [comment]
Cool! 

I like them a lot! But NodeJS theme looks mostly black & white to me.

File: material-nodejs.ini [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+models.style.italic=0
+models.style.underline=0
+
+classes.color=#FFFFFF
+classes.background=536870911
+classes.style.bold=0
+classes.style.italic=1
+classes.style.underline=0
+
+commands.color=#FFFFFFssssssssss
```
Comment (x87) [inline material-nodejs.ini]: some typo here

Comment (MatiDragon-YT) [reply material-nodejs.ini]: repaired 😅

**Comment by MatiDragon-YT** [comment]
> I like them a lot! But NodeJS theme looks mostly black & white to me.

I don't think it's the best theme to use every day.
But for memorize the high-level syntax comes in handy.

Captura de pantalla 2021-06-11 212008

**Comment by MatiDragon-YT** [comment]
1 minute

**Pull request closed without merging**